third_party/fmt-8.0.1/include/fmt/format.h

```diff
@@ -316,8 +316,10 @@ template <> constexpr auto num_bits<fallback_uintptr>() -> int {
 
 FMT_INLINE void assume(bool condition) {
   (void)condition;
-#if FMT_HAS_BUILTIN(__builtin_assume)
```

Comment:
There was a `fmt` compiler warning being emitted.  I had seen this locally for a while, but I could just look past it.  Unfortunately the Decent CI script saw this as a failure, so it was time to fix it.  There are some notes in here for the change, but basically, it seems it's a GCC false positive issue.  This brings in a cheap workaround to the warning.  I'm open to other ways to do this, but I'm not waiting to change it for this PR.  Feel free to open a new PR that reverts this and does it differently.